File: estudos/notinha.js
// se for igual ou maior que 7 = passou
// se for entre 5 e 6 = recuperação
// se for menor ou igual a 4 = não passou  

let nota1 = 7
let nota2 = 2
let nota3 = 1
let media = (nota1 + nota2 + nota3) / 3


function resultadoEscolar(media){
    if (media >= 7){
        return 'Parabéns, você passou!';
    } else if ( media >= 5 && media <= 6){
        return 'Vais precisar fazer a recuperação!';
    } else {
        return 'Infelizmente você não passou!'
    }
}

console.log(resultadoEscolar())

File: estudos/java/java/src/notinha.java
import java.util.Scanner;

public class notinha {
    public static void main(String[] args){
        Scanner leitorNotas = new Scanner(System.in);

        System.out.println("Digite sua nota do trabalho: ");
        int trabalho = leitorNotas.nextInt();

        System.out.println("Digite sua nota da prova: ");
        int prova = leitorNotas.nextInt();

        System.out.println("Digite sua nota do simulado: ");
        int simulado = leitorNotas.nextInt();

        int media = (trabalho + prova + simulado) / 3;

        if (media >= 7){
            System.out.println("Parabéns, você passou! Sua média foi: " + media);
        } else if (media >=5 && media <=6) {
            System.out.println("Você vai precisar fazer a recuperação! Sua média foi: " + media);
        } else {
            System.out.println("Infelizmente você não passou! Sua média foi: " + media);
        }
    }
}
